odoo/addons/base/models/res_company.py

```diff
@@ -61,9 +61,9 @@ def _get_default_favicon(self, original=False):
     parent_id = fields.Many2one('res.company', string='Parent Company', index=True)
     child_ids = fields.One2many('res.company', 'parent_id', string='Child Companies')
     partner_id = fields.Many2one('res.partner', string='Partner', required=True)
-    report_header = fields.Html(string='Company Tagline', help="Appears by default on the top right corner of your printed documents (report header).")
+    report_header = fields.Html(string='Company Tagline', translate=True, help="Appears by default on the top right corner of your printed documents (report header).")
```

Comment:
Hi, the ORM has a tricky feature:
the default `sanitize` attribute for HTML field is `True`
And because https://github.com/odoo/odoo/blob/f9ab5ca3e99b2883ada18a8f6deb12d45608b43f/odoo/fields.py#L1934-L1935
HTML field with `translate=True` without `sanitize=False` will be treated as `translate=html_translate` which is a model_terms translated field

The explict code would be
1. `translate=html_translate` (model_terms translation) (if all languages share the same html structure)
2. `translate=True, sanitize=False` (model translation)  (if different languages need different html structures, of course it might be risky but only admin can modify this field)